src/src/main/java/org/redfrog404/mobycraft/entity/EntityChaosMonkey.java

```diff
@@ -40,16 +38,14 @@
 	 */
 	private int maxDistance = 25;
 
-	MobycraftCommandsFactory factory = MobycraftCommandsFactory.getInstance();
-
 	public EntityChaosMonkey(World worldIn) {
 		super(worldIn);
 		this.setSize(0.5F, 0.9F);
 	}
 
 	protected void entityInit() {
 		super.entityInit();
-		this.dataWatcher.addObject(16, new Byte((byte) 0));
+		this.dataWatcher.addObject(16, (byte) 0);
```

Comment:
Here I'm removing the "boxing" of the primitive byte value in the Byte object value.  As long as folks are using Java 5 or later they should not need to box primitives.